src/tokenizer.java

```diff
@@ -21,152 +21,115 @@
      */
     public static List<token> tokenize(String filepath) throws FileNotFoundException {
         Scanner sc = new Scanner(new File(filepath));
-        int lineCount = 1;  //An incrementing line counter
+        int lineCount = 0;  //An incrementing line counter
         List<token> tokenList = new LinkedList<>(); //The list of tokens which will be returned
 
         while (sc.hasNextLine()) {  //For each line of the source file
+            lineCount++;
```

Comment:
jk, since we're incrementing at the beginning of the loop it makes sense that we start at 0. is there any particular reason this approach was chosen?

Reply:
When skipping comments, if the incrementer is at the end of the loop, it is skipped. Its not a big deal, difference between `if comment {count++;break;}` and `if comment {break;}`, but I see starting at index 0 is not intuitive since our output indexes starting at 1.